tests/sentry/nodestore/test_common.py

```diff
@@ -2,28 +2,36 @@
 Testsuite of backend-independent nodestore tests. Add your backend to the
 `ns` fixture to have it tested.
 """
-
+import pytest
+from contextlib import contextmanager
 from sentry.nodestore.django.backend import DjangoNodeStorage
-from sentry.nodestore.bigtable.backend import BigtableNodeStorage
-from tests.sentry.nodestore.bigtable.backend.tests import MockedBigtableNodeStorage
+from tests.sentry.nodestore.bigtable.backend.tests import (
+    MockedBigtableNodeStorage,
+    get_temporary_bigtable_nodestorage,
+)
 
-import pytest
+
+@contextmanager
+def nullcontext(returning):
+    # TODO: Replace with ``contextlib.nullcontext`` after upgrading to 3.7
+    yield returning
 
 
 @pytest.fixture(
     params=["bigtable-mocked", "bigtable-real", pytest.param("django", marks=pytest.mark.django_db)]
 )
```

Comment:
Tried to replace this with [pytest-cases `fixture_union`](https://smarie.github.io/python-pytest-cases/unions_theory/) to get the graph of parameterized fixtures for both `bigtable` and `django` but gave up once I realized fixture functions are globally namespaced (so you couldn't have from `tests.sentry.nodestore.bigtable.backend.tests import ns` and `tests.sentry.nodestore.django.backend.tests import ns` and got frustrated and quit.